cypress/support/awx-commands.ts

```diff
@@ -1331,6 +1332,29 @@ Cypress.Commands.add('waitForTemplateStatus', (jobID: string) => {
     });
 });
 
+Cypress.Commands.add('waitForManagementJobStatus', (jobID: string) => {
+  cy.requestGet<SystemJobTemplate>(`api/v2/system_jobs/${jobID}/`).then(
+    (response: SystemJobTemplate) => {
+      const status = response.status;
+      if (!status) {
+        cy.log('Status is null or undefined, retrying...');
+        cy.wait(100).then(() => cy.waitForManagementJobStatus(jobID));
+      } else {
```

Comment:
Do we need this `if` case if the `switch case` below has a default condition which makes the recursive call anyways?